package.json

```diff
@@ -51,9 +51,10 @@
     "lint:eslint:fix": "npm-run-all \"lint:eslint --fix\"",
     "lint:fix": "npm-run-all lint:*:fix",
     "test": "node bin/run-tests.js",
-    "test:blueprints:js": "EMBER_TYPESCRIPT_BLUEPRINTS=false pnpm test:blueprints:ts",
-    "test:blueprints:ts": "mocha node-tests/blueprints/**/*-test.js",
-    "test:blueprints": "pnpm test:blueprints:js && pnpm test:blueprints:ts",
+    "test:blueprints:js": "EMBER_TYPESCRIPT_BLUEPRINTS=false pnpm test:blueprints:transpiled-js",
+    "test:blueprints:transpiled-js": "mocha node-tests/blueprints/**/*-test.js",
+    "test:blueprints:ts": "mocha node-tests/blueprints-ts/**/*-test.js",
+    "test:blueprints": "npm-run-all test:blueprints:*",
```

Comment:
This script is used in CI and it seems to be the slowest job. We could probably split this up into separate jobs so the different blueprint tests are run in parallel which should speed things up (especially if we add more ts blueprint tests).